libs/sdk-ui-dashboard/src/presentation/export/useExportData.ts

```diff
@@ -48,3 +49,23 @@ export const useWidgetExportData = (widget: ExtendedDashboardWidget): WidgetExpo
         description: { "data-export-type": "widget-description" },
     };
 };
+
+/**
+ * @alpha
+ */
+export const useVisualizationExportData = (
+    widget: WidgetExportDataAttributes | undefined,
+    loading: boolean,
+    error: boolean,
+): WidgetExportDataAttributes | undefined => {
+    return useMemo(() => {
+        if (!widget) {
+            return undefined;
+        }
+
+        return {
+            ...widget,
+            "data-export-visualization-status": loading ? "loading" : error ? "error" : "loaded",
+        };
+    }, [widget, loading, error]);
+};
```

Comment:
Is it also used in view mode? Please check and possibly add condition for export mode here